src/libs/onyxSubscribe.js

```diff
@@ -0,0 +1,12 @@
+import Onyx from 'react-native-onyx';
+
+/**
+ * Connect to onyx data. Same params as Onyx.connect(), but returns a function to unsubscribe.
+ *
+ * @param {Object} mapping Same as for Onyx.connect()
+ * @return {function(): void} Unsubscribe callback
+ */
+export default (mapping) => {
+    const connectionId = Onyx.connect(mapping);
+    return () => Onyx.disconnect(connectionId);
+};
```

Comment:
Nice idea

Reply:
Note to myself: refactor to be a function with a name that I then default export (code style)